solutions/usaco-592.mdx

```diff
@@ -22,25 +22,36 @@ using namespace std;
 int N, A = 0;
 vector<int> positions;
 
+void setPrec() { cout << fixed << setprecision(15); }
+void unsyncIO() { cin.tie(0)->sync_with_stdio(0); }
+void setIn(string s) { freopen(s.c_str(),"r",stdin); }
+void setOut(string s) { freopen(s.c_str(),"w",stdout); }
```

Comment:
why add these if you're not going to use them?

Reply:
good question!